src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+
+}
+
+.product {
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__description {
+    padding-top: 40px;
```

Comment:
to make outer indent between elements you should use `margin`, not `padding`. fix for other cases as well

Reply:
<img width="673" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/d6a3f588-8ddb-4806-938a-cc9b33d91ce6">
still not fixed